doc/motion_planning_api/motion_planning_api_tutorial.rst

```diff
@@ -20,11 +15,11 @@ Running the Demo
 ----------------
 Open two shells. In the first shell start RViz and wait for everything to finish loading: ::
 
-  roslaunch panda_moveit_config demo.launch
+  ros2 launch moveit2_tutorials motion_planning_api.launch.py
 
 In the second shell, run the launch file: ::
 
-  roslaunch moveit_tutorials motion_planning_api_tutorial.launch
+  ros2 launch moveit2_tutorials motion_planning_api_tutorial.launch.py
```

Comment:
I actually couldn't find either of these launch files with tab-complete. I'm fairly sure I'm sourced properly. Maybe they're not installed correctly.

Reply:
Tried it again today and this worked perfectly. Nice. Please ignore.